packages/calling-stateful-client/src/OptimalVideoCountSubscriber.ts

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+/* @conditional-compile-remove(optimal-video-count) */
+import { OptimalVideoCountCallFeature } from '@azure/communication-calling';
+/* @conditional-compile-remove(optimal-video-count) */
+import { OptimalVideoCountFeatureState } from './CallClientState';
+/* @conditional-compile-remove(optimal-video-count) */
+import { CallContext } from './CallContext';
+/* @conditional-compile-remove(optimal-video-count) */
+import { CallIdRef } from './CallIdRef';
+
+/* @conditional-compile-remove(optimal-video-count) */
+/**
+ * Subscribes to a Optimal Video Count Feature events and updates the call context appropriately.
+ * @private
+ */
+export class OptimalVideoCountSubscriber {
+  private _parent: CallIdRef | 'unparented';
```

Comment:
I don't think this can be unparented. Unparented refers to this existing outside of a call (we needed this for local video streams as they exist on configuration pages). Can this be:
```suggestion
  private _callIdRef: CallIdRef;
```